pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_assigning-metadata-to-a-module.adoc

```diff
@@ -1,8 +1,18 @@
 [id="assigning-metadata-to-a-module_{context}"]
 = Assigning metadata to {ContentTerm}
+:_module-type: PROCEDURE
 
 [role="_abstract"]
-Assign metadata to {ContentTerm} to enable search engine optimization (SEO). SEO improves the findability of content. Assigned metadata also defines the URL at which the {ContentTerm} is published.
+Metadata in modules and assemblies does the following:
```

Comment:
Hey, @stoobie - we should steer away from abstract sentences along these lines. Remember that an abstract is like a summary of the module, so it needs to stand on its own as a brief description. For example "Assign metadata to your content so that Pantheon V2 can publish it under the correct product and version on the Customer Portal", etc. The line you have here is great, but probably after that.

Reply:
Thanks, @adahms. Fixed. Please approve.